src/aviary/tools/base.py

```diff
@@ -148,14 +146,12 @@ def __str__(self) -> str:
 
 
 class ToolResponseMessage(Message):
-    content: str = Field(
+    content: str | None = Field(
```

Comment:
If you look at the `description` for this field, it has to be `str` (can't be `None`).

We are going against type theory here, so we need to type ignore this one